Nguyen_Ken/Assignments/disappearing_ninjas/templates/secretNinjas.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<html lang='en'>
+<head>
+	<title>Disappearing Ninjas</title>
+	<meta charset='UTF-8'>
+	<meta name='description' content='Ninja Gold'>
+	<link rel="stylesheet" type="text/css" href="{{url_for('static', filename='css/styles.css')}}">
+</head>
+<body>
+	<div class='wrapper'>
+		<h1>No ninjas here</h1>
+
+		{% if ninja=='leonardo' %}
+			<img alt='Leonardo' src="{{url_for('static', filename='Ninjas/leonardo.jpg')}}">
+		{% elif ninja =='michelangelo' %}
+			<img alt='Michelangelo' src="{{url_for('static', filename='Ninjas/michelangelo.jpg')}}">
+		{% elif ninja =='raphael' %}
+			<img alt='Raphael' src="{{url_for('static', filename='Ninjas/raphael.jpg')}}">
+		{% elif ninja =='donatello' %}
+			<img alt='donatello' src="{{url_for('static', filename='Ninjas/donatello.jpg')}}">
+		{% elif ninja =='other' %}
+			<img alt='aprilnotapril' src="{{url_for('static', filename='Ninjas/notapril.jpg')}}">
+		{% endif %}
```

Comment:
An even cleaner way to do this:

`filename = 'Ninjas/' + ninja`

The only case you would have to look out for is notapril when ninja=other.